contrib/test_jenkins.sh

```diff
@@ -892,71 +895,82 @@ run_malloc_hook_gtest() {
 			make -C test/gtest test
 }
 
-#
-# Run the test suite (gtest)
-# Arguments: <compiler-name> [configure-flags]
-#
-run_gtest() {
-	compiler_name=$1
-
+set_common_test_flags() {
```

Comment:
3 of these flags are also gtest flags, maybe add gtest to the name

Reply:
Done